tests/everflow/test_everflow_per_interface.py

```diff
@@ -26,10 +26,18 @@
 logger = logging.getLogger(__file__)
 
 @pytest.fixture(scope="module", autouse=True)
-def skip_on_dualtor_testbed(tbinfo):
+def skip_if_not_supported(tbinfo, rand_selected_dut, ip_ver):
     if 'dualtor' in tbinfo['topo']['name']:
         pytest.skip("Skip running on dualtor testbed")
 
+    # Skip ipv6 test on Mellanox platform
+    if "mellanox" == rand_selected_dut.facts["asic_type"] and ip_ver == "ipv6":
+        pytest.skip("Match 'IN_PORTS' in EVERFLOWV6 is not supported on Mellanox platform")
+    # Skip on marvell platform
+    if "marvell" == rand_selected_dut.facts["asic_type"]:
+        pytest.skip("Match 'IN_PORTS' is not supported on Marvell platform")
```

Comment:
Thanks for the improvement. Can you refactor the code as below
```suggestion
        asic_type = rand_selected_dut.facts["asic_type"]
        unsupported_platform = ["mellanox", "marvell"]
        pytest_require(asic_type not in unsupported_platform, "Match 'IN_PORTS' is not supported on {} platform.format(asic_type))
```

Reply:
@bingwang-ms I've applied your suggestion, re-review please